boxoffice/views/admin.py

```diff
@@ -40,22 +41,37 @@ def org(organization):
     return dict(org=organization, title=organization.title)
 
 
+@app.route('/api/1/admin/o/<org>/items')
+@lastuser.requires_login
+@load_models(
+    (Organization, {'name': 'org'}, 'organization'),
+    permission='org_admin'
+    )
+@xhr_only
+@requestargs('search')
+def filter_items(organization, search=None):
+    if search:
+        filtered_items = Item.query.join(ItemCollection).filter(ItemCollection.organization == organization).filter(Item.title.ilike('%{query}%'.format(query=search))).all()
```

Comment:
Perhaps break this line up into two lines? It becomes hard to read if it's too long.